app/services/project_updater_service.rb

```diff
@@ -0,0 +1,28 @@
+class ProjectUpdaterService
+
+  def self.save(*args)
+    new(*args).save
+  end
+
+  def initialize(project, params = {})
+    @project = project
+    @params = params.to_hash
+  end
+
+  def save
+    ActiveRecord::Base.transaction do
```

Comment:
Is there some way we don't use `ActiveRecord::Base.transaction` directly here? It doesn't seem to be a good idea to couple a PORO with a Rails class.